src/irr/ext/Bullet/CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ file(GLOB_RECURSE BULLETCOLLISION_H ${BULLET_INCLUDE_PATH}/BulletCollision/*.h)
 file(GLOB_RECURSE BULLETDYNAMICS_H ${BULLET_INCLUDE_PATH}/BulletDynamics/*.h)
 file(GLOB_RECURSE LINEARMATH_H ${BULLET_INCLUDE_PATH}/LinearMath/*.h)
 
-set(IRR_EXT_INTERNAL_INCLUDE_DIR "../../../../include/irr/ext/Bullet")
+set(IRR_EXT_INTERNAL_INCLUDE_DIR "${CMAKE_SOURCE_DIR}/include/irr/ext/Bullet")
```

Comment:
IRR_ROOT_DIR not CMAKE_SOURCE_DIR

this would cause problems when a large project attempts to use our engine via `add_subdirectory`